nginx/domains/stats.ffmuc.net.conf

```diff
@@ -74,6 +74,8 @@ server {
 
         # disable compression upstream to enforce sending brotli if client allows it
         proxy_set_header   Accept-Encoding "identity";
+        proxy_set_header Upgrade $http_upgrade;
+        proxy_set_header Connection "Upgrade";
```

Comment:
It might be better to use $connection_upgrade which is defined here
https://github.com/freifunkMUC/ffmuc-salt-public/blob/eb971654439f6a44dedc931368e2253c46aadadb/nginx/domains/ffmuc.net.conf#L12-L15

To keep it clean it might be moved to some file in conf.d or even nginx.conf but I think that would be the better approach - especially as this change would alter *all* requests to upgrade which might work but which I would not expect for "regular" requests.

Reply:
It's the setting proposed by Grafana itself ...

Reply:
when I enabled it only for live API there where still errors for websockets ... so it looks like it wants it for even more requests. But feel free to fix it. I have no patience for this.

Reply:
If this is coming
```suggestion
        proxy_set_header Connection $connection_upgrade;
```